### ISSUE TITLE
Include "archived" layers

### ISSUE THREAD

**Issue by paul121** [paul121/farm_map_skagit#1]
_**TLDR:** I want to provide the ability for users to configure which layers provided by this module are displayed in farmOS maps. Such a config _could be provided with this module..._ BUT I think there could be value in a "central" config for all available farmOS map layers._

Right now this module only includes the most recent aerial imagery (2019, 2018, 2017) but many more years are available (~every two years back to 1998, 1967 & 1937). I chose to not include these additional "archive" layers in order to keep the map menu simple. BUT because these layers are available & easy to add, it would be great to include them in some way that doesn't "clutter" the map menu.

I though of providing an additional `farm_map_skagit_archive` module that could include these additional layers, but am hesitant thinking about maintenance/migration overtime.

A better option would be to _allow users to configure_ which map layers they want to be visible via a Drupal Settings Form (or similar). This way one module could provide many layers, and allow the user to configure which are displayed:

- I took a first step towards this approach by providing the available layers in `hook_farm_map_behavior_settings` rather than the `farm_map` JS behavior.
- This allows the server to potentially limit which layers are available to the behavior, and could easily be connected to a `variable` set by the Drupal System Settings form.

**BUT thinking about this further.... perhaps there should be a core farmOS setting to configure available map layers?** 

- It seems like configuration in addition to installing modules would be useful.
- This way there would be one central place to configure which map layers are available, rather than a separate configuration for each `farm_map_*` module.
- This could be a good `farm_setup` wizard step - it could display a map with ALL available layers to easily preview which each looks like.

Something to consider, though, is the reality of how many `farm_map_*` modules farms might even install. For modules providing publicly available _localized/regional_ imagery, the answer is likely 0-2, which might not warrant the need for a "central" config. But I'm curious how many imagery layers on a "larger" scale (such as NRCS Soil Survey) might become available, and be worth including in such a "central" config.
